Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/BreakStrategyTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.PlatformConfiguration.AndroidSpecific;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[NUnit.Framework.Category(Core.UITests.UITestCategories.Label)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.None, 0, "[Enhancement] Label: Break Strategy on Android", PlatformAffected.Android)]
+	public class BreakStrategyTests
+		: TestContentPage
+	{
+		protected override void Init()
+		{
+			var grid = new Grid
+			{
+				RowDefinitions = new RowDefinitionCollection()
+				{
+					new RowDefinition(), new RowDefinition()
+				}
+			};
+
+			var label = new Label
```

Comment:
Could you modify the example to test the behavior by combining different options to `BreakStrategy` and `LineBreakMode`?

Reply:
Ok, I will add that also. Thanks!

Reply:
@jsuarezruiz Added. :)